core/templates/pages/community-dashboard-page/login-required-message/login-required-message.component.spec.ts

```diff
@@ -0,0 +1,73 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for loginRequiredMessage.
+ */
+
+describe('Login required message component', function() {
+  var ctrl = null;
+  var $flushPendingTasks = null;
+  var $q = null;
+  var $scope = null;
+  var userService = null;
+
+  var mockWindow = null;
+
+  beforeEach(angular.mock.module('oppia', function($provide) {
+    mockWindow = {
+      location: {
+        reload: jasmine.createSpy('reload', () => {})
+      }
+    };
+
+    $provide.value('$window', mockWindow);
+  }));
+
+  beforeEach(angular.mock.inject(function($injector, $componentController) {
+    $flushPendingTasks = $injector.get('$flushPendingTasks');
+    var $rootScope = $injector.get('$rootScope');
+    $q = $injector.get('$q');
+    userService = $injector.get('UserService');
+
+    $scope = $rootScope.$new();
+    ctrl = $componentController('loginRequiredMessage');
+    ctrl.$onInit();
+  }));
+
+  it('should close modal with the correct value', function() {
+    expect(ctrl.OPPIA_AVATAR_IMAGE_URL).toBe(
+      '/assets/images/avatar/oppia_avatar_100px.svg');
+  });
+
+  it('should close modal with the correct value', function() {
```

Comment:
I think should redirect to login page when modal is closed?

Reply:
I forgot to change its name, I copied the unit tests structure from another file.